app/views/content/train-to-be-a-teacher/promos/_events-promo-ske.html.erb

```diff
@@ -0,0 +1,13 @@
+
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Get Into Teaching events") do %>
+  <% end %>
+  <% promo.right_section(
+        caption: "Online and in-person events",
+        heading: "Get your questions answered") do %>
+      <p>Talk to advisers and teacher training providers at a Get Into Teaching event to find out if you need to do a subject knowledge enhancement (SKE) course.</p>
```

Comment:
```suggestion
      <p>Talk to advisers and teacher training providers at a Get Into Teaching event to find out if you need to do a subject knowledge enhancement (SKE) course.</p>
```

I think this is slightly misleading - it is up to the providers you apply to as to whether you need to do SKE as a condition of offer - TTAs and the providers at the events won't be able to say definitively.

Better to say 'find out more about....' or similar?